bower.json

```diff
@@ -1,18 +1,18 @@
 {
-    "name": "pokemongogo",
-    "dependencies": {
-        "bootstrap" : "^3.3.6",
-        "jquery": "2.2.4",
-        "webcomponentsjs": "^0.7.22",
-        "web-component-tester": "^4.3.1",
-        "polymer": "Polymer/polymer#^1.5.0",
-		"app-route": "PolymerElements/app-route#^0.9.2",
-		"neon-animation": "PolymerElements/neon-animation#^1.2.4",
-		"iron-pages": "PolymerElements/iron-pages#^1.0.8",
-        "iron-ajax": "PolymerElements/iron-ajax#^1.4.3",
-        "vaadin-grid" : "^1.1.1"
-    },
-    "devDependencies": {
-        "iron-component-page": "PolymerElements/iron-component-page#^1.1.7"
-    }
+  "name": "pokemongogo",
+  "dependencies": {
+    "bootstrap": "^3.3.6",
+    "jquery": "2.2.4",
+    "webcomponentsjs": "^0.7.22",
+    "web-component-tester": "^4.3.1",
+    "polymer": "Polymer/polymer#^1.5.0",
+    "app-route": "PolymerElements/app-route#^0.9.2",
+    "neon-animation": "PolymerElements/neon-animation#^1.2.4",
+    "iron-pages": "PolymerElements/iron-pages#^1.0.8",
+    "iron-ajax": "PolymerElements/iron-ajax#^1.4.3",
+    "paper-datatable": "^0.9.8"
+  },
+  "devDependencies": {
+    "iron-component-page": "PolymerElements/iron-component-page#^1.1.7"
+  }
 }
```

Comment:
All changes in this file were the result of using the bower cli to add the paper-datatable dependency (since it was giving some errors before). We should either keep this as default or I can change it to the previous style.